src/notebooks/controllers/notebookControllerManager.ts

```diff
@@ -233,11 +233,7 @@ export class NotebookControllerManager implements INotebookControllerManager, IE
                     ? this.notebook.notebookDocuments.find((item) => item.notebookType === notebookType)
                     : undefined;
             const controller = await this.createDefaultRemoteController(notebookType, notebook);
-            if (controller) {
-                return controller;
-            }
-            traceVerbose('No default remote controller, hence returning the active interpreter');
-            return this.createActiveInterpreterController(notebookType, resource);
```

Comment:
Isn't this a breaking change for desktop?
I.e. its possible that we'll never have a controller, when in the past we'd return the active interpreter.

Reply:
whoops, I misunderstood what `isLocalLaunch` meant, good catch